### PR TITLE
Update the homepage

### LINKED ISSUE #324: Update the homepage
We need the following changes to the homepage.

### Task 1
The following changes need to be done to the description section,
Screenshot 2022-08-01 at 17 18 28

- [ ] Replace the illustration with the SL redcross logo
- [ ] Change heading style (color: `#333`, weight: `600`, size: `1.125rem`)
- [ ] Change paragraph text weight and size (weight: 400, size: `0.875rem` color: `#222`)

### Task 2

We need another section on the homepage right after the previous section named "Current Donors".
Add a dummy text for now as the description, 
Screenshot 2022-08-01 at 19 48 35

Add logos of the following organisations:
- ICRC
- IFRC 
- Singapore Redcross
- Canadian Redcross
- Quatar Red Crescent
- Microsoft
- Uber

### Assets

Redcross logo:

sl-redcross-logo

### PR DESCRIPTION
**Pull request by RandilaP**
## Purpose

The purpose of this PR is to fix #324 

## Goals
- Update home page UI
- Add current donors

## Approach
-Change fontsizes and colours
- Add redcross logo
- Add donors logos

### Screenshots
image

### CODE REVIEW

File: donor-portal/src/pages/home/index.tsx [hunk 1]
```diff
@@ -25,45 +25,86 @@ export default function Home() {
...
           </p>
           <h3>Please Donate</h3>
           <p>
-            <strong>
-              Together we can do more. We are accepting donations from both
-              organizations and individuals. Check out our aid packages below
-              and click on the Donate button to get started.
-            </strong>
+            Together we can do more. We are accepting donations from both
+            organizations and individuals. Check out our aid packages below and
```
Comment (jayasanka-sack) [inline donor-portal/src/pages/home/index.tsx:56]:
```suggestion
            organizations and individuals. Check out <Link to="/medical-needs">medical needs</Link> and
```

File: donor-portal/src/pages/home/index.tsx [hunk 2]
```diff
@@ -25,45 +25,86 @@ export default function Home() {
...
-              the Sri Lanka Red Cross along with a network of volunteers and
-              professionals to help improve the efficacy of donations. In
-              partnership with the Sri Lanka Medical Supplies Division we have
-              compiled aid packages that aggregate needs across multiple
-              hospitals to help achieve bulk price discounts from suppliers. Our
-              volunteers help negotiate prices and coordinate invoicing and
-              shipping logistics. ELIXIR facilitates transparency and
-              accountability throughout the entire supply chain.
-            </strong>
+            ELIXIR is a platform designed to meet this need. It is operated by
```
Comment (anjula-sack) [inline donor-portal/src/pages/home/index.tsx:43]: ELIXIR -> Elixir right? @jayasanka-sack

File: donor-portal/src/pages/home/styles.css [hunk 3]
```diff
@@ -76,3 +80,28 @@
...
+ max-width: 150px;
+max-height: 150px;
+padding: 3rem 3rem 3rem 3rem;
+}
+.current-donors .logo-container {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+ justify-content: center;
+align-items: center;
+}
```
Comment (anjula-sack) [inline donor-portal/src/pages/home/styles.css:107]: Add a extra line here

File: donor-portal/src/pages/home/index.tsx [hunk 4]
```diff
@@ -25,45 +25,86 @@ export default function Home() {
...
+            the Sri Lanka Red Cross along with a network of volunteers and
+            professionals to help improve the efficacy of donations. In
+            partnership with the Sri Lanka Medical Supplies Division we have
+            compiled aid packages that aggregate needs across multiple hospitals
+            to help achieve bulk price discounts from suppliers. Our volunteers
+            help negotiate prices and coordinate invoicing and shipping
+            logistics. ELIXIR facilitates transparency and accountability
```
Comment (anjula-sack) [inline donor-portal/src/pages/home/index.tsx:50]: Check the ELIXIR  case here as well.

File: donor-portal/src/pages/home/index.tsx (hunk 2 above)
Comment (jayasanka-sack) [reply donor-portal/src/pages/home/index.tsx:43]: Yes it should be "Elixir"

File: donor-portal/src/pages/home/styles.css (hunk 3 above)
Comment (jayasanka-sack) [reply donor-portal/src/pages/home/styles.css:107]: @anjula-sack It seems linter is not formatting CSS files. Could you please check it?

**Review by jayasanka-sack — COMMENTED** [review]
@RandilaP There are some pending suggestions. Could you please update the m and let me know?

File: donor-portal/src/pages/home/index.tsx (hunk 1 above)
Comment (jayasanka-sack) [reply donor-portal/src/pages/home/index.tsx:56]: You have to import `Link` as follows:

import { Link } from "react-router-dom";

**Review by jayasanka-sack — APPROVED** [review]
Thanks @RandilaP !

**Pull request merged**